src/interactive/modules.ts

```diff
@@ -48,13 +48,13 @@ export function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(statusBarItem)
 }
 
-export async function getModuleForEditor(editor: vscode.TextEditor, position: vscode.Position = editor.selection.start) {
-    let mod = manuallySetDocuments[editor.document.fileName]
+export async function getModuleForEditor(document: vscode.TextDocument, position: vscode.Position) {
+    let mod = manuallySetDocuments[document.fileName]
 
```

Comment:
```suggestion
_position = document.validatePosition(position)
if _position !== position
    position = _position
    telemetry.traceTrace(`Invalid position ${position} about to be passed to the LS; corrected to ${_position}.`)
end
```
Maybe also add validation here?

Reply:
yes, if this here doesn't help, then we should add a lot more diagnostics like that. But lets give this here one chance :)